src/themes/material/Select.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import _ from 'lodash/fp'
+import { Select as MaterialSelect, MenuItem } from '@material-ui/core'
+
+let Select = ({
+  options,
+  placeholder = 'Please Select...',
+  value = '',
+  ...props
+}) => (
+  <MaterialSelect fullWidth displayEmpty value={value} {...props}>
+    {placeholder && <MenuItem value="">{placeholder}</MenuItem>}
```

Comment:
You probably want their `displayEmpty` prop for the placeholder.

Reply:
isn't `displayEmpty` a boolean? It's already on there in the line above

Reply:
My bad. I meant `renderValue`